youtube_dl/extractor/common.py

```diff
@@ -1330,6 +1330,83 @@ def _parse_xspf(self, playlist, playlist_id):
             })
         return entries
 
+    def _download_dash_manifest(self, dash_manifest_url, video_id, fatal=True):
+        return self._download_xml(
+            dash_manifest_url, video_id,
+            note='Downloading DASH manifest',
+            errnote='Could not download DASH manifest',
+            fatal=fatal)
+
+    def _extract_dash_manifest_formats(self, dash_manifest_url, video_id, fatal=True, namespace=None, formats_dict={}):
```

Comment:
Why not just `_extract_dash_formats`? It would be more consistent with other extract formats methods. Same for all the other places in code.